opm/autodiff/GeoProps.hpp

```diff
@@ -150,7 +134,7 @@ namespace Opm
             // opmfil is hardcoded to be true. i.e the volume weighting is always used
             opmfil = true;
             if (opmfil) {
-                minPvFillProps_(grid, eclState, ntg);
+                minPvFillProps_(grid, eclState, pvol_, ntg);
```

Comment:
A separate method is usually better for readability, especially since update() is very long already.